### DISCUSSION TITLE
Mutations in Sanity Studio v3

### DISCUSSION THREAD

**Discussion by irinainina** [sanity-io/sanity#3388] [Sanity Studio Support]
Hello, I'm having trouble with Mutations in Sanity v3  
How to rewrite the example mutations proposed in the [documentation](https://www.sanity.io/docs/http-mutations) for the Sanity v3?  
Function that works in the Sanity v2 for sending a POST request looks like this
```js
const saveTitle = async (title) => {    
  const mutations = [
    {
      createOrReplace: {
        _id: '123',
        _type: 'article',
        title: title
      },
    },
  ];

  const url =
  `https://${projectId}.api.sanity.io/v2021-06-07/data/mutate/production`;

  const response = await fetch(url, {
    method: 'post',
    headers: {
      'Content-type': 'application/json',
      Authorization: `Bearer ${process.env.NEXT_PUBLIC_TOKEN}`,
    },
    body: JSON.stringify({ mutations }),
  });
  const result = await response.json();
};
```

**Comment by rexxars** [comment]
There aren't any changes to the mutation API, so this should be the same as before.

Having said that - be careful about putting tokens with write access into a public javascript bundle (such as `process.env.NEXT_PUBLIC_TOKEN`), as those tokens will be readable by anyone with access to the source of the application.
